bin/dcps_tests.lst

```diff
@@ -392,5 +392,6 @@ tests/DCPS/UnregisterType/run_test.pl: !DCPS_MIN
 
 tools/repeater/tests/basic_forward.pl: NODEJS
 
+tests/DCPS/RtpsRelay/Unit/run_test.pl: !DCPS_MIN CXX11 RTPS !NO_BUILT_IN_TOPICS !OPENDDS_SAFETY_PROFILE
```

Comment:
Remove `RTPS` from this list if it's not actually creating any RTPS discovery/transport instances